test/Decorators.test.ts

```diff
@@ -279,22 +279,32 @@ test('Handler has Body param, and is called with unexpected payload', async () =
   }
 
   //@ts-ignore
-  const { statusCode: codeBrokenJSON, body: bodyBrokenJson } = await HandlerTest.handle(eventWithBrokenBody, context)
+  const { statusCode, body } = await HandlerTest.handle(event, context)
+  expect(statusCode).toEqual(400)
+  expect(body).not.toContain('email')
+  expect(body).toContain('customBool')
+})
 
-  expect(codeBrokenJSON).toEqual(400)
-  expect(bodyBrokenJson).toContain(bodyIsNotProperJSON)
+test('Handler has Body with optional parameter, and is called with unexpected payload', async () => {
+  const message = 'okie dokie 👽'
 
-  //@ts-ignore
-  const { statusCode: statusCodeNoBody, body: bodyNoBody } = await HandlerTest.handle(eventWithNoBody, context)
+  class BodyType {
+    @IsOptional()
+    @IsEmail()
```

Comment:
it's not a problem because it's optional, but it should be `@IsInt()` 

I tried also with empty body `{}` and it's working. Fine by me even we don't use optional parameter

Reply:
I fixed the `IsEmail()`.

And `{}` sent as a body works. But the issue is that when no body is present, the handyman used to send an error message. And this made it impossible to have a body with all optional parameters work with an empty body (null / undefined)

Reply:
For me it's fine.I'm approving